src/ScriptBuilderTask.Tests.Target/AssemblyInfo.cs

```diff
@@ -5,4 +5,4 @@
     MySqlScripts = true,
     PostgreSqlScripts = true,
     OracleScripts = true,
-    ScriptPromotionPath = "$(SolutionDir)$(ProjectDir)Postfix")]
\ No newline at end of file
+    ScriptPromotionPath = @"C:\Temp\Prod")]
```

Comment:
I wonder if we should allow the user to specify arbitary variables and then have the msbuild script pass in these two. i.e. if we could say at the command line:

```
sql-persistence general assemblyname outputpath -v SolutionDir="Path\To\Solution" -v ProjectDir="Path\To\Project"
```

Reply:
Problem with this is that, if the tool is used outside the dev environment there may be no Solution/Project directory. It hinders the usage outside of the MSBuild integration.

Alternatively, the relative paths can be used, which means it'll be relative to the project directory (I think).

Reply:
Since we decided to keep the cmd tool separate to the MSBuild task, there'll be no need for this for now.